specs/Standards/MultiToken/Core.md

```diff
@@ -0,0 +1,355 @@
+
+// Returns the balances of an account for the given `token_ids`.   
+// The balances though wrapped in quotes and treated like strings, 
+// the numbers will be stored as an unsigned integer with 128 bits.
+// Arguments:
+// * `account_id`: the NEAR account that owns the tokens.
+// * `token_ids`: the tokens to retrieve the balance from
+function mt_batch_balance_of(account_id: string, token_ids: string): string[]
+
+// Returns the token supply with the given `token_id` or `null` if no such token exists.
+// The supply though wrapped in quotes and treated like a string, the number will be stored 
+// as an unsigned integer with 128 bits.
+function mt_supply(token_id: string): string | null 
+
+// Returns the token supplies with the given `token_ids`, a string value is returned or `null` 
+// if no such token exists. The supplies though wrapped in quotes and treated like strings, 
+// the numbers will be stored as an unsigned integer with 128 bits.
+function mt_batch_supply(token_ids: string[]): (string | null)[]
+```
+
+The following behavior is required, but contract authors may name this function something other than the conventional `mt_resolve_transfer` used here.
+
+```ts
+// Finalize an `mt_transfer_call` or `mt_batch_transfer_call` chain of cross-contract calls. Generically 
+// referred to as `mt_transfer_call` as it applies to `mt_batch_transfer_call` as well.
+//
+// The `mt_transfer_call` process:
+//
+// 1. Sender calls `mt_transfer_call` on MT contract
+// 2. MT contract transfers token from sender to receiver
+// 3. MT contract calls `mt_on_transfer` on receiver contract
+// 4+. [receiver contract may make other cross-contract calls]
+// N. MT contract resolves promise chain with `mt_resolve_transfer`, and may
+//    transfer token back to sender
+//
+// Requirements:
+// * Contract MUST forbid calls to this function by any account except self
+// * If promise chain failed, contract MUST revert token transfer
+// * If promise chain resolves with `true`, contract MUST return token to
+//   `sender_id`
+//
+// Arguments:
+// * `sender_id`: the sender of `mt_transfer_call`
+// * `receiver_id`: the `receiver_id` argument given to `mt_transfer_call`
+// * `token_ids`: the `token_ids` argument given to `mt_transfer_call`
+// * `amounts`: the `token_ids` argument given to `mt_transfer_call`
+// * `approved_token_ids`: if using Approval Management, contract MUST provide
+//   set of original approved accounts in this argument, and restore these
+//   approved accounts in case of revert.
+//
+// Returns total amount spent by the `sender_id`, corresponding to the `token_id`.
+// The amounts returned, though wrapped in quotes and treated like strings,
+// the numbers will be stored as an unsigned integer with 128 bits.
+// Example: if sender_id calls `mt_transfer_call({ "amounts": ["100"], token_ids: ["55"], receiver_id: "games" })`,
+// but `receiver_id` only uses 80, `mt_on_transfer` will resolve with `["20"]`, and `mt_resolve_transfer`
```

Comment:
is it spent by the receiver or sender? line 292 says sender but it would make more sense if it was the receiver

Reply:
good catch it should say receiver

Reply:
pushed up the change